local_costmap:
  global_frame: $(arg odom_frame)
  robot_base_frame: $(arg base_frame)
  update_frequency: 10.0
  publish_frequency: 5.0
  static_map: false
  rolling_window: true
  width: 6.0
  height: 6.0
  resolution: 0.05
  plugins:
    - {name: obstacle_layer, type: 'costmap_2d::ObstacleLayer'}